### PR TITLE
[doc,pwrmgr] Declare pwrmgr at D3

### LINKED ISSUE #22487: [pwrmgr] D3 Signoff

### PR DESCRIPTION
**Pull request by matutem**
The justification is added as a comment in #22487.

Fixes #22487

### CODE REVIEW

**Comment by matutem** [comment]
CHANGE AUTHORIZED: hw/ip_templates/pwrmgr/data/pwrmgr.hjson.tpl

File: hw/ip_templates/pwrmgr/doc/checklist.md [hunk 1]
```diff
@@ -95,17 +95,17 @@ Security      | [SEC_CM_COUNCIL_REVIEWED][]  | Done        |
...
+Documentation | [NEW_FEATURES_D3][]     | Done        |
+RTL           | [TODO_COMPLETE][]       | Done        |
+Code Quality  | [LINT_COMPLETE][]       | Done        |
+Code Quality  | [CDC_COMPLETE][]        | Waived      | No block-level flow available - waived to top-level signoff.
+Code Quality  | [RDC_COMPLETE][]        | Waived      | No block-level flow available - waived to top-level signoff.
+Review        | [REVIEW_RTL][]          | Done        |
+Review        | [REVIEW_DELETED_FF][]   | Done        |
+Review        | [REVIEW_SW_CHANGE][]    | Done        |
+Review        | [REVIEW_SW_ERRATA][]    | Done        |
+Review        | Reviewer(s)             | Done        | matutem@, vogelpi@
```
Comment (andreaskurth) [inline hw/ip_templates/pwrmgr/doc/checklist.md:107]: Could you please also add me (`adk@`) as a reviewer?

**Review by andreaskurth — APPROVED** [review]
Approving based on https://github.com/lowRISC/opentitan/issues/22487, though please also add me as reviewer. Thx @matutem :+1:

**Comment by andreaskurth** [comment]
CHANGE AUTHORIZED: hw/ip_templates/pwrmgr/data/pwrmgr.hjson.tpl
CHANGE AUTHORIZED: hw/top_earlgrey/ip_autogen/pwrmgr/data/pwrmgr.hjson

**Comment by matutem** [comment]
There is no way this PR can cause test failures. Merging it.

**Pull request merged**